src/pubsub/emulator.rs

```diff
@@ -126,6 +129,7 @@ impl EmulatorClient {
                 ..pubsub::api::Topic::default()
             })
             .await?;
+        debug!("Created topic: {}", topic_name);
```

Comment:
For the logged events, consider using structured fields instead of string formatting https://docs.rs/tracing/0.1.37/tracing/index.html#recording-fields

This makes it easier to consume logs programmatically because fields can be indexed/searched without searching through full logs

for example `debug!(topic = topic_name, "Created topic")` instead of `debug!("Created topic: {}", topic_name)`